web/ui/templates/rules.html

```diff
@@ -19,6 +20,7 @@ <h2>Rules</h2>
             <td style="font-weight:bold">State</td>
             <td style="font-weight:bold">Error</td>
             <td style="font-weight:bold">Evaluation Time</td>
+            <td style="font-weight:bold">Last Evaluation</td>
```

Comment:
Could you move the "Last Evaluation" before "Evaluation Time"? It would be more consistent with how we show the last scrape and scrape duration for the target.

Reply:
Updated 😃